Week1/homework/js-exercises/4-logString.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var myString ='Jalal Al Attar';
+
+console.log('the value of the string is a name, Guess the name !!' );
+console.log(myString);
+
+
+
+var myString ='David Beckham';
```

Comment:
And here the same as before, I think this will give an `already defined` error. Can you take a look at this again?

Reply:
same answer!